transforms/universal/ededup/python/README.md

```diff
@@ -13,6 +14,10 @@ The set of dictionary keys holding [EdedupPythonTransform](src/ededup_transform_
 configuration for values are as follows:
 
 * _ededup_doc_column_ - specifies the name of the column containing a document
+* _ededup_doc_id_column_ - specifies the name of the column containing a document id
+* _ededup_use_snapshot_ - specifies that ededup execution starts from a set of already seen hashes. This can be used
+  for the incremental ededup execution
+* _snapshot_directory_ - specifies a directory from which transforms are read. If this is not specified, a default
```

Comment:
I think you mean `snapshots` instead of `transforms` here.

Reply:
fixed